src/helpers/clientChronicles.js

```diff
@@ -32,7 +35,10 @@ const SUBFLOWS = new Map(Object.entries(FLOWS.reduce((acc, flow) => {
 const MAX_INACTIVITY_MS = 60 * 1000; // Minute in milliseconds.
 
 export default class ClientChronicles {
-  constructor(history, store) {
+  constructor(history) {
+    // Store is set after it is created, see setStore.
+    this.store = null;
```

Comment:
Please use connect() instead of transferring store

Reply:
This is not component, this is eventually context, e.g., global object.